channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/TelephonyReactions.kt

```diff
@@ -108,6 +113,71 @@ class TelephonyReactions(private val bargeInDefaultProps: BargeInProperties) : J
         bargeIn = BargeInProperties(mode, trigger, noInterruptTimeMs)
     }
 
+    /**
+     * This method overrides the TTS configuration of the phone channel used for the current call.
+     *
+     * Example usage:
+     * ```
+     * state("tts") {
+     *    action {
+     *        reactions.telephony?.setTtsConfig(TtsConfigAimyvoice("Никита"))
+     *    }
+     * }
+     * ```
+     * The configuration is determined by the TtsProviderConfig subclass passed in as a parameter.
+     * The appropriate provider-specific configuration settings are extracted and assigned
+     * to the corresponding fields in the TtsConfig class.
+     * @param config TtsProviderConfig subclass containing the provider-specific configuration settings to use for TTS.
+     * */
+    fun <T : TtsProviderConfig> setTtsConfig(config: T) {
+        ttsConfig = TtsConfig(
+            type = executionContext.request.telephony?.ttsConfig?.type,
```

Comment:
This looks like we're using type from original request instead of passing specific type from `config`.

Reply:
I did this because we have the ability to change the config of the current provider, but we cannot switch between providers.

Reply:
got it, thx